src/applications/vaos/new-appointment/redux/helpers/formSubmitTransformers.js

```diff
@@ -277,3 +277,72 @@ export function transformFormToAppointment(state) {
     schedulingMethod: 'direct',
   };
 }
+
+export function transformFormToVAOSCCRequest(state) {
+  const data = getFormData(state);
+  const provider = data.communityCareProvider;
+  const residentialAddress = selectVAPResidentialAddress(state);
+  const parentFacility = getChosenCCSystemId(state);
+  let practitioners = [];
+
+  if (provider?.identifier) {
```

Comment:
VAOS service requires provider selection to be on, which would mean the provider is empty or has an identifier array.